src/myframe.h

```diff
@@ -387,19 +387,79 @@ struct MyFrame : wxFrame {
                      _(L"Opens up the text from the selected cell in default application for the file type"));
 
             wxMenu *laymenu = new wxMenu();
-            MyAppend(laymenu, A_V_GS, _(L"Vertical Layout with Grid Style Rendering\tALT+1"));
-            MyAppend(laymenu, A_V_BS, _(L"Vertical Layout with Bubble Style Rendering\tALT+2"));
-            MyAppend(laymenu, A_V_LS, _(L"Vertical Layout with Line Style Rendering\tALT+3"));
+            MyAppend(laymenu, A_V_GS, 
+                #ifdef __WXMAC__

```

Comment:
#ifdef __WXMAC__
   #define CTRLORALT "ALT"
// etc
_(L"Vertical Layout with Grid Style Rendering\t" CTRLORALT  "+1")

Reply:
Ah, thanks for the input! This is far better.
Is it good now?